src/undo.h

```diff
@@ -32,7 +32,7 @@ class TxInUndoSerializer
             // Required to maintain compatibility with older undo format.
             ::Serialize(s, (unsigned char)0);
         }
-        ::Serialize(s, CTxOutCompressor(REF(txout->out)));
+        ::Serialize(s, Using<TxOutCompression>(REF(txout->out)));
```

Comment:
In commit "Convert compression.h to new serialization framework" (fd691da3b032e439936672e084b7c49086ec9ebc)

Can you drop uses of `REF` macro here and below?

Reply:
Indeed, done.